app/routes/dashboard.tsx

```diff
@@ -51,27 +53,37 @@ export default function Dashboard() {
   const { user } = useLoaderData<LoaderData>();
   const participatingExchanges = user.Participants;
   return (
-    <div>
-      <h1>Dashboard</h1>
-      <h2>Welcome {user.name}</h2>
-      <form method="post" action="/logout">
-        <button type="submit">Logout</button>
-      </form>
-      <ul>
-        {participatingExchanges.map((partExchange) => {
-          return (
-            <li key={partExchange.id}>
-              <NavLink to={partExchange.id}>
-                {partExchange.Exchange.title}
-              </NavLink>
-            </li>
-          );
-        })}
-        <li>
-          <Link to="new">New Exchange</Link>
-        </li>
-      </ul>
-      <Outlet />
-    </div>
+    <main className='grid gap-4 w-screen min-h-screen'>
+      <section className='flex flex-col gap-4 justify-center bg-red-500 px-4 sm:px-8 py-16'>
+        <SecretSantaPawsLogo className='w-3/4 sm:w-1/2' />
+        <h2 className='m-0 text-center text-white mb-4'>Welcome {user.name}</h2>
+        <ul className='flex items-center justify-center gap-2 flex-wrap m-0 p-0 list-none'>
+          {participatingExchanges.map((partExchange) => {
+            return (
+              <li key={partExchange.id} className='m-0'>
+                <NavLink to={partExchange.id} className='text-white'>
```

Comment:
One thing to be aware of with a NavLink. It will automatically add an `active` class to the active link based on the location. So we just need to make sure we have a style defined for that state. On quick search, I didn't see one.

Reply:
Ah, ok, got ya. I wasn't totally sure what to do about those links, if we move the Logout button to the header then maybe those two links can look more like secondary buttons and sit side by side?

Reply:
Yeah. That sounds great. It's mostly just a way for them to see other exchanges if they happen to be part of multiple. Like maybe one from last year. Exchanges can be archived. Haven't done much with that.

Reply:
I've refactored the Header. Just gotta remember to add it to each page that you'd like it. The default Header only contains the GitHub logo. In the Dashboard i pass the links and logout button as children.